composer.json

```diff
@@ -1,101 +1,102 @@
 {
-    "name": "griidc/pelagos",
-    "license": "BSD",
     "type": "project",
-    "minimum-stability": "dev",
-    "prefer-stable": true,
-    "autoload": {
-        "psr-4": {
-            "": "src/"
-        },
-        "files": [
-            "app/AppKernel.php"
-        ]
-    },
-    "autoload-dev": {
-        "psr-4": {
-            "Tests\\": "tests/",
-            "": "tests/helpers"
-        }
-    },
+    "license": "proprietary",
     "require": {
-        "php": ">=7.1",
-        "symfony/symfony": "3.4.*",
-        "doctrine/orm": "^2.5",
+        "php": "^7.1.3",
+        "ext-ctype": "*",
+        "ext-iconv": "*",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/doctrine-cache-bundle": "^1.2",
-        "symfony/swiftmailer-bundle": "^2.3",
-        "symfony/monolog-bundle": "^3.0",
-        "sensio/distribution-bundle": "^5.0",
-        "sensio/framework-extra-bundle": "^3.0.2",
-        "incenteev/composer-parameter-handler": "^2.0",
-        "slim/slim": "2.*",
-        "slim/views": "0.1.*",
-        "indigophp/doctrine-annotation-autoload": "^0.1.0@dev",
-        "friendsofsymfony/rest-bundle": "^1.7",
-        "jms/serializer-bundle": "^1.1",
-        "nelmio/api-doc-bundle": "^2.11",
         "doctrine/doctrine-fixtures-bundle": "^2.3",
-        "willdurand/hateoas-bundle": "^1.1",
-        "simplethings/entity-audit-bundle": "^1.0",
-        "friendsofsymfony/jsrouting-bundle": "^1.6",
-        "php-amqplib/rabbitmq-bundle": "^1.9",
+        "doctrine/doctrine-migrations-bundle": "^1.2",
+        "doctrine/orm": "^2.5",
+        "friendsofsymfony/elastica-bundle": "^5.1.1",
+        "friendsofsymfony/jsrouting-bundle": "^2.1",
+        "friendsofsymfony/rest-bundle": "^2.5",
+        "guzzlehttp/guzzle": "~6.0",
+        "hylianshield/base32-crockford": "^1.0",
+        "jms/serializer-bundle": "^2.0",
         "jsor/doctrine-postgis": "^1.1",
+        "kriswallsmith/buzz": "0.16.*",
```

Comment:
There is a version 1 now that is stable. Suggest we test this with version ^1.0

Reply:
We should probably get rid of this library and use Guzzle instead, we are using it only in Datastore class

Reply:
We'll leave it for a refactor then.